service-base/src/main/java/org/eclipse/hono/service/metric/MetricsTags.java

```diff
@@ -418,4 +422,18 @@ public static Tags forService(final String name) {
     public static Tag getTenantTag(final String tenant) {
         return Tag.of(MetricsTags.TAG_TENANT, Strings.isNullOrEmpty(tenant) ? "UNKNOWN" : tenant);
     }
+
+    /**
+     * Creates a tag for a tenant identifier.
```

Comment:
The _JavaDoc_  mistakenly describes about the _tenant identifier_  instead of the _cipher suite_.

Reply:
Thanks for spotting :-)